xkit.js

```diff
@@ -12,6 +12,8 @@ var xkit_global_start = Date.now();  // log start timestamp
 			standard:
 				window.window === window.top &&
 				document.location.href.indexOf("://www.tumblr.com/dashboard/iframe?") === -1,
+			form_frame:
+				document.location.href.indexOf("://www.tumblr.com/neue_web/iframe/") !== -1,
```

Comment:
How confident are we in this url's stability?

Reply:
i'm 100% confident that the neue web iframes will disappear before their URLs are changed

Reply:
👍